toolkit/tools/imagecustomizerapi/script.go

```diff
@@ -10,20 +10,22 @@ import (
 type Script struct {
 	// Path is the path of the script file.
 	// Mutually exclusive with 'Content'.
-	Path string `yaml:"path"`
+	Path string `yaml:"path" json:"path,omitempty"`
 	// Content is the inline string content of the script to run.
 	// Mutually exclusive with 'Path'.
-	Content string `yaml:"content"`
+	Content string `yaml:"content" json:"content,omitempty"`
 	// Interpreter is the name or path of the process that will execute the script.
 	// When 'Content' is specified, the default value is '/bin/sh'.
 	// When 'Path' is specified and 'interpreter' is not specified, then the script file is executed directly.
-	Interpreter string `yaml:"interpreter"`
+	Interpreter string `yaml:"interpreter" json:"interpreter,omitempty"`
 	// Arguments is a list of additional arguments to pass to the script.
-	Arguments []string `yaml:"arguments"`
+	Arguments []string `yaml:"arguments" json:"arguments,omitempty"`
 	// EnvironmentVariables are a set of environment variables to set when executing the script.
-	EnvironmentVariables map[string]string `yaml:"environmentVariables"`
+	EnvironmentVariables map[string]string `yaml:"environmentVariables" json:"environmentVariables,omitempty"`
 	// Name is an optional value used to reference the script in the logs.
-	Name string `yaml:"name"`
+	Name string `yaml:"name" json:"name,omitempty"`
+
+	SHA256Hash string `yaml:"sha256hash" json:"sha256hash,omitempty"`
```

Comment:
I would remove `yaml:"sha256hash"`. That way, the user can't accidentally provide the value in the config file.

Reply:
removed